src/main/bash/envs/kobman-tobdflow.sh

```diff
@@ -2,13 +2,13 @@
 
 function __kobman_development_greenlight_dir {
```

Comment:
What does the function name __kobman_development_greenlight_dir suggest?

Reply:
This function name can be modified as per suggestions, the name is suppose to give the idea that "greenlight development environment directory " creation is happening here